webdriver/tests/interaction/element_clear.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success, assert_dialog_handled
+from tests.support.inline import inline
+
+
+def clear(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/clear"
+                                  .format(session_id=session.session_id,
+                                          element_id=element))
+
+
+# 14.2 Element Clear
+
+def test_no_browsing_context(session, create_window):
+    # 14.2 step 1
+    session.window_handle = create_window()
+    session.close()
+
+    result = clear(session, "box1")
+    assert_error(result, "no such window")
+
+
+def test_element_not_found(session):
+    # 14.2 Step 2
+    result = clear(session, "box1")
+    assert_error(result, "no such element")
+
+
+def test_element_not_editable(session):
+    # 14.2 Step 3
+    session.url = inline("""        
+        <p id="para" contenteditable="false">This is not an editable paragraph.</p>
+        """)
```

Comment:
Then the `contenteditable="false"` attribute is pointless.  Also you can drop the ID and the multiline Python string for readability.

Reply:
deleted